desktop/apps/artwork/components/commercial/templates/price.jade

```diff
@@ -1,5 +1,7 @@
-- var showPriceLabel = artwork.price && artwork.availability !== 'sold'
-if artwork.sale_message
+- var isPermanentCollection = artwork.availability === 'permanent collection'
+- var isOnLoan = artwork.availability === 'on loan'
+- var showPriceLabel = artwork.price && artwork.availability !== 'sold' && !isOnLoan
+if artwork.sale_message || isPermanentCollection
```

Comment:
`Permanent Collection` availability does not end up in `sale_message` ?

Reply:
It does not, since in the artwork bricks we don't want to show it. It's just on the artwork page.